.github/workflows/snapshot-build.yml

```diff
@@ -81,7 +81,7 @@ jobs:
     strategy:
       matrix:
         build:
-          - os: ubuntu-20.04
+          - os: ${{ '$GITHUB_ACTOR' == 'subspace' && 'self-hosted-ubuntu' || 'ubuntu-22.04' }}
```

Comment:
I have left comment during previous review already that this shouldn't be 22.04, this should be 20.04 instead or you break 20.04 compatibility in new releases.